test/libman.IntegrationTest/CliBaseTest.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.Web.LibraryManager.Cli.IntegrationTest;
+
+[TestClass]
+[DeploymentItem(@"TestPackages", "TestPackages")]
+public class CliTestBase
+{
+    private const string CliPackageName = "Microsoft.Web.LibraryManager.Cli";
+    private const string ToolInstallPath = "./TestInstallPath";
+    private const string ManifestFileName = "libman.json";
+    private string _testDirectory;
+
+    [TestInitialize]
+    public async Task TestInitialize()
+    {
+        _testDirectory = Path.Combine(Path.GetTempPath(), "LibmanTest" + Guid.NewGuid().ToString());
+        Directory.CreateDirectory(_testDirectory);
+
+        // create an empty nuget.config with only our package source
+        await RunDotnetCommandLineAsync("new nuget.config");
```

Comment:
Would prefer `new nugetconfig`.  Both nuget.config and nugetconfig work as the template name, but all the docs point to nugetconfig (one word, no dot)

Reply:
easily done :)